src/handlers/gestures/gesture.ts

```diff
@@ -25,28 +25,28 @@ export type GestureType =
   | BaseGesture<ForceTouchGestureHandlerEventPayload>
   | BaseGesture<NativeViewGestureHandlerPayload>;
 
-export type GestureRef = number | GestureType | React.RefObject<GestureType>;
+export type GestureRef =
+  | number
+  | GestureType
+  | React.RefObject<GestureType | undefined>
+  | React.RefObject<any>; // allow adding a ref to a gesture handler
```

Comment:
I wonder if we can constrain it to the React components instead of `any` here.

Reply:
I think it's a good idea, done in https://github.com/software-mansion/react-native-gesture-handler/pull/1736/commits/256f5f4362c3f9d9810e66ad4ed073a7117b1c06.